client/src/components/Home/styles.css

```diff
@@ -0,0 +1,44 @@
+.searchbar{
```

Comment:
This css file is currently not being used. I have kept it as a reference for structure for making UX in the future.